src/main/java/com/google/graphgeckos/dashboard/fetchers/github/CommitData.java

```diff
@@ -0,0 +1,87 @@
+package com.google.graphgeckos.dashboard.fetchers.github;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Map;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CommitData {
+
+  // Name of the branch.
+  private String branch;
+
+  /**
+   * Extracts the name of the branch from the reference.
+   * @param ref Git reference.
+   */
+  @JsonProperty("ref")
+  private void extractBranch(String ref) {
+    String[] refComponents = ref.split("/");
+    branch = refComponents[refComponents.length - 1];
+  }
+
+  // Git commit hash.
+  private String commitHash;
+
+  // Time of the Git commit push.
+  private String timestamp;
+
+  /**
+   * Extracts commit id (also known as commit hash) and timestamp nested fields from the data of the head_commit
```

Comment:
Comments get more helpful if they are not direct translation of the code, but focus on a more natural language explanation or add context that isn't part of the code. The portion that the id is the commit hash for example is such a bit. Otherwise you can probably try to condense this comment a little.

Reply:
Done!